omod/src/main/webapp/resources/scripts/conditionlist/controllers/addcondition.controller.js

```diff
@@ -78,12 +78,18 @@ function ConditionController($scope, RestfulService, ConditionModel, ConceptMode
 
     self.getSelectedDate = self.getSelectedDate || function () {
             var datePicker = angular.element(document.getElementsByName('conditionStartDate'))[0];
-            return datePicker.value;
+            var date = datePicker.value;
+            var formattedDate =date.replace(/-/g,'/');
+            var dateWithNoTime = new Date(formattedDate);
+            return dateWithNoTime.toUTCString();
         }
 
     self.getEndDate = self.getEndDate || function() {
             var endDatePicker = angular.element(document.getElementsByName('conditionEndDate'))[0];
-            return endDatePicker.value;
+            var date = endDatePicker.value;
+            var formattedDate =date.replace(/-/g,'/');
+            var dateWithNoTime = new Date(formattedDate);
+            return dateWithNoTime.toUTCString();       
```

Comment:
Are you changing the return value from Date to String?

Reply:
Yes. When the date is converted to UTCString then the backend (server-side) there is no further manipulation to change the time to the local timezone of the server.